### ISSUE TITLE
ADF Beacon Refresh

### ISSUE THREAD

**Issue by DWAC-GakkSimian** [ciribob/DCS-CTLD#82]
Not sure if DCS changed in 2.7 or 2.8.  I vaguely recall the ADF needle pointing consistently.
Now it only points when the beacon.ogg file is running.

The issue is that the hard-coded refresh cycle in ctld.refreshRadioBeacons() was 30 seconds and beacon.ogg runs for 31 seconds.  So an odd sync issue happened for me.

I added a global var ctld.beaconRefreshPeriod = 65 to allow the refresh to wait till the longest ogg file (beaconsilent.ogg) completed and use that in the function.  Now the ADF needle deflects for the ogg run period and waits till refresh to run again.  Seems to be usable.

**Comment by davidp57** [comment]
What are you asking for in this issue?

**Comment by davidp57** [comment]
@DWAC-GakkSimian can you answer the question please? Else I'm going to delete the issue...

**Comment by DWAC-GakkSimian** [comment]
I'm rather surprised that nobody has verified or refuted this claim.  That the ODB's get very quirky.

My fix is to ensure that the play length of the beacon.ogg file is shorter than the expected transmit time.  I merely mentioned it here as a possible thing to fix for future releases.  Feel free to drop this if you wish.

**Comment by davidp57** [comment]
Thanks for your answer @DWAC-GakkSimian; so, if I understand correctly, the fix needed is to add a specific delay to the ADF beacon loop that is greater that the beacon sound file length?

**Comment by DWAC-GakkSimian** [comment]
Yes, or comments explaining.  IOW, when the beacon.ogg is running, there is a needle deflection.  The delay should be configurable up front (in script) with an explanation that the number exceed the play-time of the ogg file.  Hope I'm being clear.
